inspirehep/dojson/hep/rules/bd9xx.py

```diff
@@ -247,8 +248,13 @@ def references2marc(self, key, value):
         a_list.append('hdl:{}'.format(pis_value)
                       if pis_schema is 'HDL' else pis_value)
 
-    a_list = filter(lambda x: x is not None, a_list)
     arxiv = get_value(reference, 'arxiv_eprint')
+    arxiv = 'arXiv:{}'.format(arxiv) if arxiv else None
+    report_n = get_value(reference, 'report_number')
+    r_list = [arxiv, report_n]
+
+    a_list = filter(lambda x: x is not None, a_list)
+    r_list = filter(lambda x: x is not None, r_list)
```

Comment:
As already mentioned, this filtering for empty values is done by doJSON, no need to do it yourself.